### ISSUE TITLE
Proxy-switcher 0.3.6 settings for other (non-HTML) protocols do not persist after a browser restart

### ISSUE THREAD

**Issue by ArcticHaze** [rNeomy/proxy-switcher#54]
I have the same proxy set up for HTML, SSL and FTP. When I set it up, all three lines in the Firefox (60.0.1, 64 -bit on Win7) are correctly set to my proxy server IP and port numbers. However, after restarting Firefox, only the HTML line is filled up while the other two become empty. The result is I cannot use https sites using the proxy (this is how I found the problem in the first place).

Happens: every time. 

The expected result: SSL proxy setting persists after restarting Firefox.

**Comment by ArcticHaze** [comment]
Yes, resetting all the network.proxy.backup values (they were 0s) did make it work for me, at least after the first Firefox restart.

By the way, the backup values were a leftover from the time I struggled with the previous bug :(

I believe you can close this bug for now but if this happens again, you will here from me ;)

**Comment by svscorp** [comment]
It also happens in my case. In addition to the behavior you described, it says '20:29:46: Cannot set this proxy type!'
